client_interface.py

```diff
@@ -52,32 +52,26 @@ def send_to_server(self, message):
                     # Cycle "for" is finished when client disconnect from server
                     message = msg.json(loads=json.loads)
                     if "game_state" in message:
-                        self.set_game_state(message)
-                    elif "robot_data" in message:
-                        self.set_robot_data(message)
+                        self.set_game_state(message, robot_name)
```

Comment:
Pokud pošleš klientovi nejprve stav hry, a až potom info o robotovi, tento kód nezafunguje. Nastavení stavu hry teď závisí na tom, že známe jméno robota, nemělo by asi tedy být ve dvou větvích if/elif (pak je zde šance, že to někdy nevyjde). Nevím ale, jak to vylepšit :(

Reply:
ale já první vždy posílám clientovi robot_name